### ISSUE TITLE
NVDA alpha snapshot update check failed

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#17868]
### Steps to reproduce:
1. Install the NVDA Alpha35749.
2. Press NVDA+N,H,C.
3. Check: https://download.nvaccess.org/snapshots/alpha/
4. Check: https://download.nvaccess.org/snapshots/index.html
### Actual behavior:
Perform step 2: No new version was found;
Perform step 3: List the latest version as alpha-35749;(2025-03-20 23:39:29)
Perform step 4: List the latest version as alpha-35158

### Expected behavior:
NVDA should check the new version, and we merged multiple PRs after 2025-03-20 23:39:29.
For the URLs listed in Steps 3 and 4, please confirm which channel is the correct one for downloading the NVDA Alpha version and make them work as expected.
### NVDA logs, crash dumps and other attachments:
n/a
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-35749
#### Windows version:
Windows10
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by SaschaCowley** [comment]
@cary-rowen  thanks for bringing this to our attention. We're investigating.

**Comment by josephsl** [comment]
Hi,

Looks like this is solved - alpha-35820,0fc7254e was offered on systems with older alph builds instlaled.

Thanks.

**Comment by michaelDCurran** [comment]
Yes, snapshots are being published again, as the related issue with NV Access's infrastructure was fixed.

**Comment by cary-rowen** [comment]
Hi @michaelDCurran 

For the URLs listed in Steps 3 and 4, please confirm which channel is the correct one for downloading the NVDA Alpha version.

**Comment by SaschaCowley** [comment]
@cary-rowen you want https://download.nvaccess.org/snapshots/alpha/. The "NVDA Development Snapshots" page is currently not being updated.
